<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">div {
	background-image: url(:/img/background.png);
	opacity = 07.5;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_monitoring">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(178, 130, 176);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									stop: 0 #9b75f6, stop: 1 #b89cfc);
}</string>
    </property>
    <property name="text">
     <string>Отобразить текущие параметры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Чтобы запустить автоматический полив,отключите плату от USB и подключите к батарейке</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_manual_irrigation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Проверка насоса&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(178, 130, 176);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									stop: 0 #9b75f6, stop: 1 #b89cfc);
}</string>
    </property>
    <property name="text">
     <string>Начать полив вручную, 2 сек</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_last_irrigation">
       <property name="text">
        <string>Время с последнего полива, час</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_soil_humidity"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_air_temp">
       <property name="text">
        <string>Температура воздуха, ˚C</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_last_irrigation"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_air_temp"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_soil_humidity">
       <property name="text">
        <string>Влажность почвы, %</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_air_humidity">
       <property name="text">
        <string>Влажность воздуха, %</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_light">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_light">
       <property name="text">
        <string>Освещённость, %</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_air_humidity"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_time_to_future_irrigation">
       <property name="text">
        <string>Время до ближайшего полива, час</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_time_to_future_irrigation"/>
     </item>
    </layout>
   </widget>
   <zorder>pushButton_monitoring</zorder>
   <zorder>label</zorder>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_start_manual_irrigation</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
